chpl/chpl-service/src/main/java/gov/healthit/chpl/util/CertificationResultRules.java

```diff
@@ -98,13 +103,34 @@ public CertificationResultRules() {
                     }
                 }
             }
-        } catch (final ParserConfigurationException pce) {
+        } catch (ParserConfigurationException pce) {
             LOGGER.error(pce.getMessage(), pce);
-        } catch (final SAXException se) {
+        } catch (SAXException se) {
             LOGGER.error(se.getMessage(), se);
-        } catch (final IOException ioe) {
+        } catch (IOException ioe) {
             LOGGER.error(ioe.getMessage(), ioe);
         }
+
+        // update svap to use this thing too
```

Comment:
Want to update this constructor to be more clean code-y? The one has just become/already was really legthy. Something like (or I'm sure you or Todd could come up with something even better):

`//move above code into a method, hopefully will delete the method one day
initializeRulesFromFile();
//move below (new) code into a method)
initializeRulesFromDatabase();
`